File: C++/chanukah.cpp
#include <iostream>
using namespace std;

int main() {
  int cases;
  cin >> cases;

  for (int i = 1; i <= cases; i++) {
    int k, n;
    cin >> k >> n;

    cout << k << " " << ((n * (n + 1) / 2) + n) << endl;
  }
}

File: Java/chanukah.java
import java.io.*;
import java.util.*;

public class chanukah {

  public static void main(String[] args) throws IOException {
    BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
    int cases = Integer.parseInt(br.readLine());

    for (int i = 1; i <= cases; i++) {
      StringTokenizer st = new StringTokenizer(br.readLine());
      int k = Integer.parseInt(st.nextToken());
      int n = Integer.parseInt(st.nextToken());

      System.out.println(k + " " + ((n * (n + 1) / 2) + n));
    }
  }
}
